### PR TITLE
[add] context manager to switch company

### PR DESCRIPTION
**Pull request by simahawk**

### CODE REVIEW

File: anthem/__init__.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Copyright 2016 Camptocamp SA
 # License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0.en.html)
 
-__version__ = "0.9.0.dev0"
+__version__ = "0.9.0"
```
Comment (guewen) [inline anthem/__init__.py]: Shouldn't it be changed when we do the release?

Comment (simahawk) [inline anthem/__init__.py]: you're right but I expect you to relase it before holidays: today ;p

**Comment by simahawk** [comment]
@guewen ping :)

**Comment by guewen** [comment]
@simahawk released

**Comment by simahawk** [comment]
:+1:

**Pull request merged**